kraken-build/src/kraken/std/python/tasks/pytest_task.py

```diff
@@ -61,10 +61,20 @@ def get_execute_command_v2(self, env: MutableMapping[str, str]) -> list[str] | T
                 DeprecationWarning,
             )
 
+        if (
+            self.settings.source_directory not in tests_dir
+            and self.settings.source_directory not in self.include_dirs.get()
+            and list((self.project.directory / self.settings.source_directory).rglob("*_test.py"))
```

Comment:
Do you need to search for `test_*.py` too?

Reply:
When tests are located under `src/` the goal is to have them nearby the relevant code. Therefore, the `test` is added as a suffix and not a prefix to preserve the alphabetical ordering.

Reply:
Conventionally, yes, but pytest's test discovery supports either. I don't mind us not supporting this, so long as it's a conscious choice based on looking at Kraken-using codebases rather than an oversight.

Reply:
This is on purpose!